src/nidcpower/metadata/enums.py

```diff
@@ -300,27 +300,33 @@
     'Event': {
         'values': [
             {
-                'name': 'NIDCPOWER_VAL_SOURCE_COMPLETE',
```

Comment:
I think these changes are OK with my limited nimi-python knowledge. Looks like it still marks the 'python_name' with what it previously had for 'name' so those should line up during nimi-python codegen but I'm not 100% sure.